### DISCUSSION TITLE
Why the move to non-open-source (non-commercial)?

### DISCUSSION THREAD

**Discussion by liquidat** [wg-easy/wg-easy#764] [General]
Hi, I just realized that the license of wg-easy was changed to be CC-BY-NC-SA. This seems to be a drastic change, after the recent move to AGPL.

Is there any more information why this was done? This move effectively makes this non-open source.

**Comment by peterlewis** [comment]
Thank you for bringing up this question. I want to provide some clarification regarding the reversion from AGPL that you mentioned. The reversion was present because there was an unintentionally-included commit to the master branch, which should have undergone squashing before merging.

After this, the entire pull request (https://github.com/wg-easy/wg-easy/pull/749) was reverted (https://github.com/wg-easy/wg-easy/pull/753) because the intended license was not properly represented in the merged content.

Regarding the decision to switch to the CC BY-NC-SA 4.0 license, it was proposed by @WeeJeWel. This license aligns closely with the project's intention of not being used in commercial products and also provides the benefits of a more formal licensing model, while still resembling the previous informal license.

I appreciate your feedback, and I encourage you to discuss this further with @WeeJeWel. As @pheiduck and I are not authorized to make governance decisions beyond those approved by @WeeJeWel, we focus on improving the technical aspects of the application.

Thank you once again for raising this question.

**Comment by WeeJeWel** [comment]
It’s still open source. The source is open. This is a better license than it was. If you want to embed this for commercial use, contact me for a license.

**Comment by liquidat** [comment]
@peterlewis Thanks for the insights. Sad to read it, but it is your code, so you do as it works best for you.

@WeeJeWel 

>  It’s still open source.

No. The term "open source" was coined by the OSI, their definition is widely accepted, and "non-commercial" is contrary to open source. You are absolutely free in choosing the license that you want for your code - but please don't try to mislabel things.
